ukrainian_word_stress/compile_dict.py

```diff
@@ -0,0 +1,164 @@
+import marisa_trie
+import csv
+import sys
+import collections
+import logging
+import tqdm
+
+from ukrainian_word_stress.tags import compress_tags
+
+
+log = logging.getLogger(__name__)
+
+
+ACCENT = '\u0301'
+VOWELS = "уеіїаояиюєУЕІАОЯИЮЄЇ"
+
+def compile(csv_path: str) -> marisa_trie.BytesTrie:
+    trie = []
+    by_basic = _parse_dictionary(csv_path)
+    for basic, forms in by_basic.items():
+        accents_options = len(set(form for form, _ in forms))
+        if accents_options == 1:
+            # no need to store tags if there's no ambiguity
+            value = accent_pos(forms[0][0])
+        else:
+            value = b''
+            for form, tags in forms:
+                pos = accent_pos(form)
+                value += pos + b'^' + compress_tags(tags) + b'$'
+        trie.append((basic, value))
+    return marisa_trie.BytesTrie(trie)
+
+
+def _parse_dictionary(csv_path):
+    by_basic = collections.defaultdict(list)  # TODO: change to set
+    skipped = 0
+    for row in tqdm.tqdm(csv.DictReader(open(csv_path))):
```

Comment:
You probably want to use this
https://github.com/tqdm/tqdm#redirecting-logging

Reply:
I just removed the logging under tqdm :)